targets/unity-v2/source/Authentication/Strategies/UsernamePasswordAuthStrategy.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using PlayFab.ClientModels;
+using UnityEngine;
+
+namespace PlayFab.Authentication.Strategies
+{
+    internal sealed class UsernamePasswordAuthStrategy : SilentAuthStrategy
+    {
+        public override void Authenticate(PlayFabAuthService authService, Action<LoginResult> resultCallback, Action<PlayFabError> errorCallback)
+        {
+            if (string.IsNullOrEmpty(authService.Email) || string.IsNullOrEmpty(authService.Password))
+            {
+                authService.InvokeDisplayAuthentication();
+                return;
+            }
+
+            base.Authenticate(authService, silentCallback =>
```

Comment:
rename silentCallback to resultCallback so that it matches the other authStrategies and doesn't confuse me.